web/experimental/embed-new.html

```diff
@@ -28,6 +28,7 @@
     <link href="../packages/codemirror/addon/hint/show-hint.css" rel="stylesheet" media="screen">
     <link href="../packages/codemirror/theme/elegant.css" rel="stylesheet" media="screen">
     <script src="../packages/codemirror/codemirror.js" defer></script>
+    <script type="text/javascript" src="packages/split/split.min.js"></script>
```

Comment:
@jacob314 - given that people are relying on package:split for the js resources, we may consider also publishing the dart code (see https://github.com/dart-lang/dart-pad/pull/1036/files#diff-a9e9818d8a5762e16905000b338d8a9c above).

@johnpryan, if that sounds good, you might track that as a new issue against the devtools repo.

Reply:
good idea, created https://github.com/flutter/devtools/issues/565 to track